### PR TITLE
[NOTASK] several liquidity management and API adjustments

### PR DESCRIPTION
**Pull request by TonySemikin**

### CODE REVIEW

File: src/subdomains/staking/application/services/liquidity-management.service.ts [hunk 1]
```diff
@@ -78,15 +79,19 @@ export class LiquidityManagementService {
 
   // --- MASTERNODES ---- //
   private async createMasternodes(count: number): Promise<void> {
+    const idleMasternodes = await this.masternodeService.getIdleMasternodes(count);
```
Comment (Kolibri1990) [inline src/subdomains/staking/application/services/liquidity-management.service.ts:82]: The comment from line 52 should be here or?

File: src/integration/masternode/application/services/masternode.service.ts [hunk 2]
```diff
@@ -61,6 +61,10 @@ export class MasternodeService {
     return this.masternodeRepo.find();
   }
 
+  async getIdleMasternodes(count: number): Promise<Masternode[]> {
+    return this.masternodeRepo.find({ where: { state: MasternodeState.IDLE }, take: count });
```
Comment (TonySemikin) [inline src/integration/masternode/application/services/masternode.service.ts:65]: @Kolibri1990 what is the scenario if there is not enough idle masternodes (output count != input count)?

Comment (Kolibri1990) [reply src/integration/masternode/application/services/masternode.service.ts]: Just talked.

File: src/subdomains/staking/application/services/liquidity-management.service.ts (hunk 1 above)
Comment (TonySemikin) [reply src/subdomains/staking/application/services/liquidity-management.service.ts:82]: done!

**Review by Kolibri1990 — APPROVED** [review]

**Pull request merged**